### PR TITLE
added follow us section

### PR DESCRIPTION
**Pull request by AswaniBolisetti**
@MitulSonagara 
I have added follow us section successfully
kindly check it out

follow2

follow3

### CODE REVIEW

**Comment by AswaniBolisetti** [comment]
@MitulSonagara could you please review and merge it

**Comment by MitulSonagara** [comment]
@AswaniBolisetti 
Please resolve this error

image

**Comment by AswaniBolisetti** [comment]
@MitulSonagara 
could you please provide more details like when did you get this error

**Comment by MitulSonagara** [comment]
@AswaniBolisetti 

When i check the deployment of the website...

File: src/app/(app)/page.tsx [hunk 1]
```diff
@@ -5,8 +5,13 @@ import { Button } from "@/components/ui/button";
 import { ChevronRight, Eye, Shield, Zap } from "lucide-react";
 import Link from "next/link";
 import { getServerSession } from "next-auth";
-import ScrollToTopButton from "@/components/ScrollToTopButton"; // Import the new component
-import ScrollProgressBar from "@/components/ScrollProgressBar"; // Import the progress bar
+import Image from "next/image";
+
```
Comment (MitulSonagara) [inline src/app/(app)/page.tsx:9]: Here scrollProgressbar is missing

File: src/app/layout.tsx [hunk 2]
```diff
@@ -35,10 +37,13 @@ export default async function RootLayout({
               defaultTheme="system"
               enableSystem={true}
             >
-              {/* <Navbar/> */}
+               {/* Include Sidebar here */}
               {children}
               <Toaster richColors expand={true} />
-              <Modals />
+
+              <UsernameChangeForm />
```
Comment (MitulSonagara) [inline src/app/layout.tsx:44]: And why there is UsernameChangeForm.

**Review by MitulSonagara — CHANGES_REQUESTED** [review]

**Comment by AswaniBolisetti** [comment]
@MitulSonagara 
kindly close this issue, I found my mistake, I'll raise this issue again and create another pull request by considering all these issues

**Comment by MitulSonagara** [comment]
@AswaniBolisetti 

Okay no problem...

**Pull request closed without merging**